src/main/java/roomescape/reservation/service/ReservationService.java

```diff
@@ -0,0 +1,160 @@
+package roomescape.reservation.service;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import roomescape.common.exception.DuplicateException;
+import roomescape.common.exception.InvalidIdException;
+import roomescape.common.exception.InvalidTimeException;
+import roomescape.common.exception.message.IdExceptionMessage;
+import roomescape.common.exception.message.ReservationExceptionMessage;
+import roomescape.member.domain.Member;
+import roomescape.member.domain.repository.MemberRepository;
+import roomescape.reservation.domain.Reservation;
+import roomescape.reservation.domain.repository.ReservationRepository;
+import roomescape.reservation.dto.MyReservationResponse;
+import roomescape.reservation.dto.ReservationResponse;
+import roomescape.reservation.dto.admin.AdminReservationRequest;
+import roomescape.reservation.dto.admin.AdminReservationSearchRequest;
+import roomescape.reservation.dto.user.UserReservationRequest;
+import roomescape.reservationTime.domain.ReservationTime;
+import roomescape.reservationTime.domain.respository.ReservationTimeRepository;
+import roomescape.theme.domain.Theme;
+import roomescape.theme.domain.repository.ThemeRepository;
+
+@Service
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+    private final ReservationTimeRepository timeRepository;
+    private final ThemeRepository themeRepository;
+    private final MemberRepository memberRepository;
+
+    public ReservationService(
+            ReservationRepository reservationRepository,
+            ReservationTimeRepository timeRepository,
+            ThemeRepository themeRepository,
+            MemberRepository memberRepository
+    ) {
+        this.reservationRepository = reservationRepository;
+        this.timeRepository = timeRepository;
+        this.themeRepository = themeRepository;
+        this.memberRepository = memberRepository;
+    }
+
+    public List<ReservationResponse> findAll() {
+        return reservationRepository.findAll().stream()
+                .map(ReservationResponse::from)
+                .toList();
+    }
+
+    public List<MyReservationResponse> findAllByMemberId(final Long memberId) {
+        return reservationRepository.findAllByMemberId(memberId).stream()
+                .map(MyReservationResponse::from)
+                .toList();
+    }
+
+    public List<ReservationResponse> findAllByMemberAndThemeAndDate(
+            final AdminReservationSearchRequest adminReservationSearchRequest
+    ) {
+        Long memberId = adminReservationSearchRequest.memberId();
+        Long themeId = adminReservationSearchRequest.themeId();
+        LocalDate dateFrom = adminReservationSearchRequest.dateFrom();
+        LocalDate dateTo = adminReservationSearchRequest.dateTo();
```

Comment:
```suggestion
    public List<ReservationResponse> findAllByMemberAndThemeAndDate(
            final AdminReservationSearchRequest request
    ) {
        Long memberId = request.memberId();
        Long themeId = request.themeId();
        LocalDate dateFrom = request.dateFrom();
        LocalDate dateTo = request.dateTo();
```

변수명이 단순해도 의미를 전달하기에 충분해 보이는데 어떻게 생각하시나요?

Reply:
아 네, 굳이 변수명을 길게 쓸 필요가 없겠네요. 리팩토링 했습니다!